### ISSUE TITLE
CSS background-image not rendered

### ISSUE THREAD

**Issue by dplong** [arachnys/athenapdf#135]
I have several .png files referenced in my styles.css file like this:
> background-image: url("/img/box.png");

However, athenapdf does not render them. Is this a known problem? There is just a blank area where the image should appear.

**Comment by MrSaints** [comment]
It should render with a background image by default. Try setting `-Z 3` or higher. It might be related to the scaling. I am assuming you are trying to convert a site on `localhost` ?

**Comment by dplong** [comment]
That didn't help. The same HTML renders correctly with wkhtmltopdf. This is the command I used:
> sudo docker run --network="host" --security-opt seccomp:unconfined --rm -v $(pwd):/converted/ arachnysdocker/athenapdf athenapdf -Z 3 http://localhost:3001/test.html test.pdf

**Comment by MrSaints** [comment]
Out of curiosity, a. how are you linking the stylesheet? and b. can I have a look at your stylesheet?

Finally, can you try using the following:

```
docker run --network="host" --rm -v $(pwd):/conversions/ -w /conversions/ arachnysdocker/athenapdf:3 http://localhost:3001/test.html test.pdf
```

**Comment by dplong** [comment]
athenapdf:3 didn't help. I'll see about putting together an isolated stylesheet and HTML that exhibits the problem.

**Comment by sanohin** [comment]
Fixed in #181 .
